apps/backend/src/tests/response_api.test.ts

```diff
@@ -0,0 +1,555 @@
+// /* eslint-disable no-underscore-dangle */ // Need to access httpResponse._body
+// // import { Neighborhood, User } from '@prisma/client';
+import { Response } from 'supertest';
+import app from '../app';
+import seed from './seed';
+import testHelpers from './testHelpers';
+import { LoginData, UpdateResponseData } from '../types';
+
+const supertest = require("supertest"); // eslint-disable-line
+// 'require' was used because supertest does not support import
+
+const api = supertest(app);
+
+const ANTONINAS_LOGIN_DATA: LoginData = {
+  username: 'antonina',
+  password: 'secret',
+};
+
+const BOBS_LOGIN_DATA: LoginData = {
+  username: 'bob1234',
+  password: 'secret',
+};
+
+const ANTONINAS_NHOOD_ID = 2;
+const RADUS_REQUEST_ID = 2;
+const ANTONINAS_RESPONSE_ID = 1;
+const ANTONINAS_USER_ID = 2;
+
+const loginUser = async (loginData: LoginData) => {
+  const loginResponse = await api.post('/api/login').send(loginData);
+
+  return loginResponse;
+};
+
+beforeAll(async () => {
+  await seed();
+});
+
+// We have ANTONINAS_RESPONSE (Antonina's user_id = 2, response_id = 1)
+// The response is associated with RADUS_REQUEST (request_id = 2
+// created by the user RADU, user_id = 4)
+// RADUS_REQUEST is in ANTONINAS_NEIGHBORHOOD (neighborhood_id = 2, admin_id = 2)
+
+describe('Tests for updating a request: PUT /responses/:id', () => {
+  let token: string;
+
+  beforeAll(async () => {
+    const loginResponse: Response = await loginUser(ANTONINAS_LOGIN_DATA);
+    token = loginResponse.body.token;
+  });
+
+  beforeEach(async () => {
```

Comment:
do we need this before each test?

Reply:
I thought that was the SOP. We login user in `beforeAll` and seed data in `beforeEach`. Feel free to make changes if you something else makes more sense.